model_servers/whispercpp/Makefile

```diff
@@ -17,4 +17,5 @@ all: build download-model-whisper-small run
 .PHONY: download-model-whisper-small # small .bin model type testing
 download-model-whisper-small:
 	cd ../../models && \
-	make MODEL_NAME=ggml-small.bin MODEL_URL=https://huggingface.co/ggerganov/whisper.cpp/resolve/main/ggml-small.bin -f Makefile download-model
+	make download-model-whisper-small
```

Comment:
I think we remove this and instead in the `model_servers/common/Makefile.common` we add:
```Makefile
.PHONY: download-model-mistral
download-model-mistral:
	$(MAKE) -f  ../../models/Makefile download-model-mistral
```